riff-raff.yaml

```diff
@@ -91,7 +91,7 @@ deployments:
         ImagingAmiId:
           BuiltBy: amigo
           AmigoStage: PROD
-          Recipe: grid-imaging
+          Recipe: grid-imaging-eoan
```

Comment:
Can I suggest avoiding adding distro names to the AMIgo recipe as it makes the next upgrade harder:
- Find current recipe
- Clone settings
- Bake
- Raise PR
- Delete old recipe

An alternative would be to update the base image on the `grid-imaging` recipe and bake.

Reply:
This will need upgrading at the end of April when the next LTS is released. At that point I'll update grid-imaging to then new LTS and delete this recipe. I want to keep both recipes around until we're confident that the error rate changes in the new graphicsmagick (discussed in planning) doesn't effect us.